misc/python/materialize/mzcompose/services/materialized.py

```diff
@@ -94,6 +94,13 @@ def __init__(
             *DEFAULT_CRDB_ENVIRONMENT,
         ]
 
+        if image == "materialize/materialized:v0.110.3":
+            print("HERE!")
+            image = "materialize/materialized:devel-abf064c9dd95566ee13f8053c1b287f9f220d591"
+        if image == "materialize/materialized:v0.109.1":
+            print("THERE!")
+            image = "materialize/materialized:devel-f16c0c1242775d8a579cc68684ef01f3b8f80101"
+
```

Comment:
I assume you didn't mean to include this here!

Reply:
Removed this and marked the updated PR for automerge.

Reply:
Oops, of course. Thank you.